radio/src/targets/horus/mavlink_telem_driver.cpp

```diff
@@ -0,0 +1,158 @@
+/*
+ * Copyright (C) OpenTX
+ *
+ * Based on code named
+ *   th9x - http://code.google.com/p/th9x
+ *   er9x - http://code.google.com/p/er9x
+ *   gruvin9x - http://code.google.com/p/gruvin9x
+ *
+ * License GPLv2: http://www.gnu.org/licenses/gpl-2.0.html
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ */
+
+#include "opentx.h"
+
+#if defined(RADIO_T16) && defined(PCBX10) && defined(MAVLINK_TELEM)
```

Comment:
is it possible to have only #if defined(MAVLINK_TELEM) or better TELEMETRY_MAVLINK here, it's better to avoid to check the radio everywhere in the code. Then if we want to add this option to another radio, we only have to edit board.h (or better CMakeLists.txt to enable the feature). I think CMakeLists.txt is better because you use some RAM here, I am not sure everyone will like this option on radios which have too little RAM available

Reply:
done. thx.